api/src/main/java/org/openmrs/module/ugandaemrreports/library/HIVPatientDataLibrary.java

```diff
@@ -92,7 +89,7 @@ public PatientDataDefinition getCurrentViralLoad() {
     }
 
     public PatientDataDefinition getCurrentRegimen() {
-        return df.getObsByEndDate(hivMetadata.getCurrentRegimen(), null, TimeQualifier.LAST, df.getObsValueCodedConverter());
+        return df.getObsByEndDate(hivMetadata.getCurrentRegimen(), null, TimeQualifier.LAST, new ObsValueConverter());
```

Comment:
@solemabrothers Why the change from df.getConverter to new Converter?

Reply:
This is because the old converter would return Concept IDs instead of Values for Current Regimen in a CSV format. The new format works for both Excel and CSV. The same applies for dates

Reply:
@solemabrothers so why not also remove the old convertor since its no longer used? So that the code is clean everywhere